stock_move_location/wizard/stock_move_location.xml

```diff
@@ -0,0 +1,171 @@
+        <field name="model">wiz.stock.move.location</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet>
+                    <div class="oe_button_box" name="button_box" />
+                    <div>
+                        <label for="edit_locations">
+                            Edit Locations
+                        </label>
+                        <field name="edit_locations" widget="boolean_toggle" />
+                    </div>
+                    <div>
+                        <label for="exclude_reserved_qty">
+                            Exclude Reserved Quantities
+                        </label>
+                        <field name="exclude_reserved_qty" widget="boolean_toggle" />
+                    </div>
+                    <div groups="stock.group_stock_multi_locations">
+                        <label for="apply_putaway_strategy">
+                Apply putaway strategy for moving products
+            </label>
+                        <field name="apply_putaway_strategy" widget="boolean_toggle" />
+                    </div>
+                    <group name="picking_type">
+                        <field name="picking_type_id" />
+                    </group>
+                    <group name="main">
+                        <field name="origin_location_disable" invisible="True" />
+                        <field
+                            name="origin_location_id"
+                            readonly="origin_location_disable"
+                        />
+                        <field name="destination_location_disable" invisible="True" />
+                        <field
+                            name="destination_location_id"
+                            readonly="destination_location_disable"
+                        />
+                        <field name="company_id" invisible="1" />
+                    </group>
+                    <group name="lines">
+                        <field
+                            name="stock_move_location_line_ids"
+                            nolabel="1"
+                            mode="tree,kanban"
+                            colspan="2"
+                        >
+                            <tree
+                                editable="bottom"
+                                decoration-info="move_quantity != max_quantity"
+                                decoration-danger="(move_quantity &lt; 0) or (move_quantity > max_quantity)"
+                                create="0"
+                            >
+                                <field
+                                    name="product_id"
+                                    domain="[('type','=','product')]"
+                                />
+                                <field
+                                    name="product_uom_id"
+                                    string="UoM"
+                                    groups="uom.group_uom"
```

Comment:
I get an error, steps to reproduce in runbot:

- login
- go to "Inventory > Operations > Move from location"
- select Origin Location: WH/Stock
- select Destination Location: WH/Stock/Shelf 1
- push Button: Immediate Transfer

Result:
```
Traceback (most recent call last):
  File "/opt/odoo/odoo/http.py", line 1764, in _serve_db
    return service_model.retrying(self._serve_ir_http, self.env)
  File "/opt/odoo/odoo/service/model.py", line 133, in retrying
    result = func()
  File "/opt/odoo/odoo/http.py", line 1791, in _serve_ir_http
    response = self.dispatcher.dispatch(rule.endpoint, args)
  File "/opt/odoo/odoo/http.py", line 1995, in dispatch
    result = self.request.registry['ir.http']._dispatch(endpoint)
  File "/opt/odoo/odoo/addons/base/models/ir_http.py", line 222, in _dispatch
    result = endpoint(**request.params)
  File "/opt/odoo/odoo/http.py", line 741, in route_wrapper
    result = endpoint(self, *args, **params_ok)
  File "/opt/odoo/addons/web/controllers/dataset.py", line 24, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/opt/odoo/addons/web/controllers/dataset.py", line 20, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/opt/odoo/odoo/api.py", line 468, in call_kw
    result = _call_kw_multi(method, model, args, kwargs)
  File "/opt/odoo/odoo/api.py", line 453, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/opt/odoo/addons/web/models/models.py", line 73, in web_save
    self = self.create(vals)
  File "<decorator-gen-12>", line 2, in create
  File "/opt/odoo/odoo/api.py", line 414, in _model_create_multi
    return create(self, [arg])
  File "/opt/odoo/odoo/models.py", line 4615, in create
    records = self._create(data_list)
  File "/opt/odoo/odoo/models.py", line 4862, in _create
    field.create([
  File "/opt/odoo/odoo/fields.py", line 4332, in create
    self.write_batch(record_values, True)
  File "/opt/odoo/odoo/fields.py", line 4358, in write_batch
    self.write_real(records_commands_list, create)
  File "/opt/odoo/odoo/fields.py", line 4547, in write_real
    flush()
  File "/opt/odoo/odoo/fields.py", line 4503, in flush
    comodel.create(to_create)
  File "<decorator-gen-12>", line 2, in create
  File "/opt/odoo/odoo/api.py", line 415, in _model_create_multi
    return create(self, arg)
  File "/opt/odoo/odoo/models.py", line 4615, in create
    records = self._create(data_list)
  File "/opt/odoo/odoo/models.py", line 4877, in _create
    records._validate_fields(name for data in data_list for name in data['stored'])
  File "/opt/odoo/odoo/models.py", line 1459, in _validate_fields
    check(self)
  File "/mnt/data/odoo-addons-dir/stock_move_location/wizard/stock_move_location_line.py", line 62, in _constraint_max_move_quantity
    self._compare(record.move_quantity, record.max_quantity, rounding) == 1
  File "/mnt/data/odoo-addons-dir/stock_move_location/wizard/stock_move_location_line.py", line 55, in _compare
    return float_compare(qty1, qty2, precision_rounding=precision_rounding)
  File "/opt/odoo/odoo/tools/float_utils.py", line 169, in float_compare
    rounding_factor = _float_check_precision(precision_digits=precision_digits,
  File "/opt/odoo/odoo/tools/float_utils.py", line 29, in _float_check_precision
    assert precision_rounding is None or precision_rounding > 0,\
AssertionError: precision_rounding must be positive, got 0.0

The above server error caused the following client error:
RPC_ERROR: Odoo Server Error
    RPC_ERROR
        at makeErrorFromResponse (http://oca-stock-logistics-warehouse-17-0-pr2053-fe9c3d3456ab.runboat.odoo-community.org/web/assets/3c8b26e/web.assets_web.min.js:2886:163)
        at XMLHttpRequest.<anonymous> (http://oca-stock-logistics-warehouse-17-0-pr2053-fe9c3d3456ab.runboat.odoo-community.org/web/assets/3c8b26e/web.assets_web.min.js:2890:13)
```

The reason of this error is the `groups="uom.group_uom"` in line 67, that prevents the onchange to work properly.

A quick and simple solution would be to change this line into `groups="base.group_user"`.

A better solution would be to convert the onchange into a computed method.